engine/src/cmdss.cpp

```diff
@@ -111,8 +111,14 @@ Parse_stat MCGo::parse(MCScriptPoint &sp)
 	Chunk_term lterm = CT_UNDEFINED;
 
 	initpoint(sp);
-	if (sp.skip_token(SP_FACTOR, TT_PROPERTY, P_INVISIBLE) == PS_NORMAL)
+	if (sp.skip_token(SP_FACTOR, TT_PROPERTY, P_INVISIBLE) == PS_NORMAL) {
```

Comment:
Can you put the opening parentheses here and below on the following line please?

Reply:
Now no longer necessary - with an enum rather than bools the if becomes a single line again.